v3/plugins/redhat/php/1.3.11/meta.yaml

```diff
@@ -12,7 +12,7 @@ category: Language
 firstPublicationDate: "2020-05-26"
 spec:
   containers:
-    - image: "quay.io/eclipse/che-sidecar-php:7-fc377b6"
+    - image: "quay.io/eclipse/che-sidecar-php:7-9854327"
```

Comment:
do we know why php is requiring nodejs ?

Reply:
I'm not sure TBH, it's been there from the beginning: https://github.com/che-dockerfiles/che-sidecar-php/commit/295ec3296ce7720511e98d5f4736f0ad8028ee98

Reply:
looks like it's a leftover when we had to have nodejs for che-theia endpoint but not related directly to this PR

Reply:
Alright, I'll remove node as the base image and use an alpine image base instead